core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -131,9 +131,9 @@
   public static final String MESSAGE_ACCEPT_ENCODING = "grpc-accept-encoding";
 
   /**
-   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 100 MiB.
+   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 4 MiB.
    */
-  public static final int DEFAULT_MAX_MESSAGE_SIZE = 100 * 1024 * 1024;
+  public static final int DEFAULT_MAX_MESSAGE_SIZE = 4 * 1024 * 1024;
```

Comment:
I don't think we have our heart set on it too much, but I would generally prefer base 2 simply because any such number in base 10 is annoying to take into account. The number of zeros at the end make it hard to read anyway.